devicesetup/src/main/java/io/particle/android/sdk/devicesetup/ui/DiscoverDeviceActivity.java

```diff
@@ -342,34 +383,46 @@ private void onMaxAttemptsReached() {
     }
 
     private void onDeviceClaimedByOtherUser() {
-        String dialogMsg = getString(R.string.dialog_title_owned_by_another_user,
-                getString(R.string.device_name), sparkCloud.getLoggedInUsername());
-
-        new Builder(this)
-                .setTitle(getString(R.string.change_owner_question))
-                .setMessage(dialogMsg)
-                .setPositiveButton(getString(R.string.change_owner),
-                        (dialog, which) -> {
-                            dialog.dismiss();
-                            log.i("Changing owner to " + sparkCloud.getLoggedInUsername());
+        log.i("Changing owner to " + sparkCloud.getLoggedInUsername());
 //                        // FIXME: state mutation from another class.  Not pretty.
 //                        // Fix this by breaking DiscoverProcessWorker down into Steps
-                            resetWorker();
-                            discoverProcessWorker.needToClaimDevice = true;
-                            discoverProcessWorker.gotOwnershipInfo = true;
-                            discoverProcessWorker.isDetectedDeviceClaimed = false;
-                            DeviceSetupState.deviceNeedsToBeClaimed = true;
-
-                            showProgressDialog();
-                            startConnectWorker();
-                        })
-                .setNegativeButton(R.string.cancel,
-                        (dialog, which) -> {
-                            dialog.dismiss();
-                            startActivity(new Intent(DiscoverDeviceActivity.this, GetReadyActivity.class));
-                            finish();
-                        })
-                .show();
+        resetWorker();
+        discoverProcessWorker.needToClaimDevice = true;
+        discoverProcessWorker.gotOwnershipInfo = true;
+        discoverProcessWorker.isDetectedDeviceClaimed = false;
+        DeviceSetupState.deviceNeedsToBeClaimed = true;
+
+        showProgressDialog();
+        startConnectWorker();
+
+//        String dialogMsg = getString(R.string.dialog_title_owned_by_another_user,
```

Comment:
Unused I can remove, but I thought I would leave it in case we ever want to add it back